net/vnstat2/patches/100-noexit.patch

```diff
@@ -58,3 +58,27 @@
  		} else if (strcmp(argv[currentarg], "--initdb") == 0) {
  			s->initdb = 1;
  			s->showhelp = 0;
+--- a/src/dbsql.c
++++ b/src/dbsql.c
+@@ -976,7 +976,7 @@ int db_insertdata(const char *table, con
+ 
+ int db_removeoldentries(void)
+ {
+-	int rc, i;
++	int rc, i, dbifcount;
+ 	char sql[256];
+ 	const char *datatables[] = {"fiveminute", "hour", "day", "month", "year"};
+ 	const int32_t entrylimits[] = {cfg.fiveminutehours * 12, cfg.hourlydays * 24, cfg.dailydays, cfg.monthlymonths, cfg.yearlyyears};
+@@ -987,8 +987,11 @@ int db_removeoldentries(void)
+ 		printf("db: removing old entries\n");
+ 	}
+ 
+-	if (db_getiflist(&dbifl) <= 0) {
++	dbifcount = db_getiflist(&dbifl);
```

Comment:
please add a simple description for this change in your commit message:
why is this required?

Reply:
I extended the commit message to mention why this is necessary so the noexit patch keeps working.

Reply:
Thank you!